build/shakaBuildHelpers.py

```diff
@@ -296,7 +296,7 @@ def get_node_binary(module_name, bin_name=None):
     package_data = json.load(open_file(json_path, 'r'))
     bin_data = package_data['bin']
 
-    if type(bin_data) is str or type(bin_data) is unicode:
+    if type(bin_data) is str or type(bin_data) is str:
```

Comment:
This doesn't make sense.  This is "if X or X", where both X's are "type(bin_data) is str".

Reply:
For Python 2, we need to check for the unicode type as well as str.  For Python 3, we only need to check for str.

Reply:
Looking more closely, get_node_binary dates from a time before I knew what "npx" was.  This whole mess can be replaced with "npx".  We will take care of that.